### PR TITLE
Modify force type casting finder

### PR DESCRIPTION
**Pull request by lukyanoffpashok**

### CODE REVIEW

File: aibolit/patterns/force_type_casting_finder/force_type_casting_finder.py [hunk 1]
```diff
@@ -20,47 +20,19 @@
...
-        for item in items:
-            if (item['line']) is not None:
-                last_line_number = item['line']
-                continue
-            item['line'] = last_line_number
-
-        return items
-
     def value(self, filename: str):
         ''''''
```
Comment (aravij) [inline aibolit/patterns/force_type_casting_finder/force_type_casting_finder.py:30]: redundant
```suggestion
```

File: aibolit/patterns/force_type_casting_finder/force_type_casting_finder.py [hunk 2]
```diff
@@ -20,47 +20,19 @@
...
-
-        for item in items:
-            if (item['line']) is not None:
-                last_line_number = item['line']
-                continue
-            item['line'] = last_line_number
-
-        return items
-
     def value(self, filename: str):
```
Comment (aravij) [inline aibolit/patterns/force_type_casting_finder/force_type_casting_finder.py:29]: Needs return type.

File: aibolit/patterns/force_type_casting_finder/force_type_casting_finder.py [hunk 3]
```diff
@@ -20,47 +20,19 @@
...
-        list_tree = self.__tree_to_list(tree)
-        num_str = []
-        for node in list_tree:
-            if node['ntype'] == javalang.tree.Cast:
-                k = int(node['line'])
-                if k not in num_str:
-                    num_str.append(k)
-        return num_str
+        tree = AST(build_ast(filename))
+        lines = []
```
Comment (aravij) [inline aibolit/patterns/force_type_casting_finder/force_type_casting_finder.py:32]: Empty list needs to be specified with types
```suggestion
        lines: List[int] = []
```

Comment (lukyanoffpashok) [reply aibolit/patterns/force_type_casting_finder/force_type_casting_finder.py]: ok

File: aibolit/patterns/force_type_casting_finder/force_type_casting_finder.py (hunk 2 above)
Comment (lukyanoffpashok) [reply aibolit/patterns/force_type_casting_finder/force_type_casting_finder.py]: ok

File: aibolit/patterns/force_type_casting_finder/force_type_casting_finder.py (hunk 1 above)
Comment (lukyanoffpashok) [reply aibolit/patterns/force_type_casting_finder/force_type_casting_finder.py]: ok

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#453` is not assigned, can't get performer

**Pull request merged**